kinesis-to-http/zoo-stats-api-graphql_staging.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "zoo-stats-api-graphql_staging",
+  "description": "It forwards Kinesis stream to Caesar staging",
```

Comment:
still a reference to caesar on this line

Reply:
Yeh forgot to save before commiting...